extras/nest-config.in

```diff
@@ -66,7 +66,7 @@ while test $# -gt 0; do
         echo "@ALL_CXXFLAGS@"
         ;;
     --libs)
-        echo "-L$prefix/@CMAKE_INSTALL_LIBDIR@ @ALL_LIBS@"
+        echo "-L$prefix/@CMAKE_INSTALL_LIBDIR@/nest @ALL_LIBS@"
```

Comment:
Shouldn't there be a similar change for the `--libdir` switch in [line 91](https://github.com/nest/nest-simulator/pull/1513/files#diff-07b895965dd0b3a757363159d0466805R91) of this file?

Reply:
The behaviour is presently consistent: e.g. the MyModule cmake file takes the ``${CMAKE_INSTALL_LIBDIR}`` (equal to ``nest-config --libdir``) and again appends the ``nest`` subdirectory.

Reply:
OK. Then maybe we should not change that. Thanks for the explanation.